sdcm/cluster.py

```diff
@@ -3222,36 +3230,35 @@ def _parse_cs_results(lines):
     def get_stress_results_bench(self, queue):
         results = []
         ret = []
-        self.log.debug('Wait for %s bench stress threads results', queue[TASK_QUEUE].qsize())
+        self.log.debug('Wait for %s bench stress threads results', queue[TASK_QUEUE].qsize())  # pylint: disable=no-member
         queue[TASK_QUEUE].join()
         while not queue[RES_QUEUE].empty():
             results.append(queue[RES_QUEUE].get())
 
-        for node, result in results:
+        for _, result in results:
```

Comment:
`results.values()`

Reply:
this is not a dict.. it's a list of tuples